### DISCUSSION TITLE
Lab Assignment #2 Submission

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/21S-ASIAAM-191A#89] [Assignments]
Please use this thread to post your individual lab assignments, which is the GitHub pages URL.

For example:
https://albertkun.github.io/21S-ASIAAM-191A-Assignments/Week_02/index.html

**Comment by ghost** [comment]
https://ktgreenler.github.io/21S-ASIAAM-191A-Assignments/Week_02/index.html

**Comment by rachan2023** [comment]
https://rachan2023.github.io/21S-ASIAAM-191A-Assignments/Week_02/index.html

**Comment by KenNL42** [comment]
https://kennl42.github.io/21S-ASIAAM-191A-Assignments/Week_02/index.html

**Comment by rh5140** [comment]
https://rh5140.github.io/21S-ASIAAM-191A-Assignments/Week_02/index.html

**Reply by alizezila** [reply]
The dolphin markers are awesome!!

**Reply by alizezila** [reply]
And your art is so cool 😳

**Reply by rh5140** [reply]
Thank you! I really like the colors you chose for your website, and a map of the concerts you've been to is such a fun idea! I don't know much about the live music scene though, so I can't guess which one you crowd-surfed at...

**Comment by alizezila** [comment]
https://alizezila.github.io/21S-ASIAAM-191A-Assignments/Week_02/index.html

**Comment by skarakas1** [comment]
https://skarakas1.github.io/21S-ASIAAM-191A-Assignments/Week_02/index.html

**Comment by okcirE** [comment]
https://okcire.github.io/21S-ASIAAM-191A-Assignments/Week_02/index.html

**Comment by nnhimichelle** [comment]
https://nnhimichelle.github.io/21S-ASIAAM-191A-Assignments/Week_02/index.html

**Comment by latishasumardy** [comment]
https://latishasumardy.github.io/21S-ASIAAM-191A-Assignments/Week_02/index.html

**Comment by Wny-Duong** [comment]
https://wny-duong.github.io/21S-ASIAAM-191A-Assignments/Week_02/index.html

**Comment by gu-ana** [comment]
https://gu-ana.github.io/21S-ASIAAM-191A-Assignments/Week_02/index.html

**Comment by vcta99** [comment]
https://vcta99.github.io/21S-ASIAAM-191A-Assignments/Week_02/index.html

**Comment by jsxu315** [comment]
https://jsxu315.github.io/21S-ASIAAM-191A-Assignments/Week_02/index.html

**Comment by jason-muljadi** [comment]
https://jason-muljadi.github.io/21S-ASIAAM-191A-Assignments/Week_02/

**Comment by epham36** [comment]
https://epham36.github.io/21S-ASIAAM-191A-Assignments/Week_02/

**Comment by avaitt** [comment]
https://avaitt.github.io/21S-ASIAAM-191A-Assignments/Week_02/index.html

**Comment by lvalles10** [comment]
https://lvalles10.github.io/21S-ASIAAM-191A-Assignments/Week_02/index.html

**Comment by annyrviloria** [comment]
https://annyrviloria.github.io/21S-ASIAAM-191A-Assignments/Week_02/index.html

**Comment by brillianz** [comment]
https://brillianz.github.io/21S-ASIAAM-191A-Assignments/Week_02/index.html

**Comment by ninalong** [comment]
https://ninalong.github.io/21S-ASIAAM-191A-Assignments/Week_02/index.html

**Comment by anmindi** [comment]
https://anmindi.github.io/21S-ASIAAM-191A-Assignments/Week_02/index.html

**Reply by anmindi** [reply]
http://127.0.0.1:5500/Week_02/index.html
